pkg/api/util/util.go

```diff
@@ -45,10 +51,7 @@ func SetAuthToken(config model.Reader) error {
 	defer tokenLock.Unlock()
 
 	// Noop if token is already set
-	if initializedBy != "" {
-		if initializedBy != "SetAuthToken" {
-			log.Infof("function SetAuthToken was called after CreateAndSetAuthToken was called")
-		}
+	if initSource != uninitialized {
```

Comment:
I think it makes sense to use the same style of switch statement being used in `CreateAndSetAuthToken`, in order to log an error when this method gets called after the authToken has already been created.

Reply:
Ultimately, I believe it is not necessary to log when `SetAuthToken` is called after `CreateAndSetAuthToken` because  `CreateAndSetAuthToken` function is basically a superset of `SetAuthToken`.
In the opposite, wanting to create the auth_token after having it already creating it is more problematic.
What is your opinion ?